ethcore/types/src/client_types.rs

```diff
@@ -62,6 +62,27 @@ pub struct ClientReport {
 	pub gas_processed: U256,
 	/// Memory used by state DB
 	pub state_db_mem: usize,
+	/// I/O statistics for state DB.
```

Comment:
```suggestion
	/// I/O statistics for the state DB.
```

Reply:
…but is it true that it is only for state? I don't think so?

Reply:
> …but is it true that it is only for state? I don't think so?

Yes, it is
https://github.com/openethereum/openethereum/pull/11523/files#diff-40c7c65265e511d07158f62712f69573R1101

Reply:
Hmm, I see. :/ So we miss some IO from other sources then?